sdk/managednetwork/Microsoft.Azure.Management.ManagedNetwork/src/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure ManagedNetwork Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure ManagedNetwork management functions for managing the Microsoft Azure ManagedNetwork service.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.0.1")]
+[assembly: AssemblyFileVersion("1.0.0.1")]
```

Comment:
Please do not change `AssemblyVersion` and use `1.0.1.0` as `AssemblyFileVersion`.
Just FYI, `AssemblyVersion` only needs to be changed when there is a major version update.